exporter/signalfxexporter/internal/translation/delta_translator.go

```diff
@@ -20,10 +20,15 @@ func newDeltaTranslator(ttl int64, done chan struct{}) *deltaTranslator {
 		sweepIntervalSeconds = 1
 	}
 	m := ttlmap.New(sweepIntervalSeconds, ttl, done)
-	m.Start()
 	return &deltaTranslator{prevPts: m}
 }
 
+func (t *deltaTranslator) start() {
+	if t.prevPts != nil {
+		t.prevPts.Start()
```

Comment:
This should be the equivalent functionality as before, just starting when `start` is called, instead of when the object is created. This is a better UX, from my understanding.